### PR TITLE
Homework 20

### PR DESCRIPTION
**Pull request by Bilyk-VM**
Створення і наповнення баз данних

### CODE REVIEW

File: 20/homework_20.2 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+     [
```
Comment (klymenok) [inline 20/homework_20.2]: This won't work in the SQL console

File: 20/homework_20.4.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import sqlite3
+
+with sqlite3.connect("homework20.sqlite") as connection:
+    cursor = connection.cursor()
+    query = ('CREATE TABLE IF NOT EXISTS users ('
+             'id INTEGER PRIMARY KEY,'
+             'first_name TEXT UNIQUE,'
```
Comment (klymenok) [inline 20/homework_20.4.py]: `UNIQUE` here doesn't allow to add a first_name that already exists. For example, `John Snow` and `John Walker` are different names, but your table doesn't allow to add another John here.

To check for unique combinations you can use UNIQUE on the table level and set columns that should be unique together:
```
...
first_name TEXT,
last_name TEXT,
age INTEGER,
UNIQUE(first_name, last_name))

**Review by klymenok — CHANGES_REQUESTED** [review]
Check why files from 12 and 14 homework are here

File: 20/homework_20.4.py (hunk 2 above)
Comment (klymenok) [reply 20/homework_20.4.py]: @Bilyk-VM you changed UNIQUE constraint in _20.4 file but didn't change it here as well

**Comment by Bilyk-VM** [comment]
Комітив і випадково замість homework_20, написав  homework_22. Як змінити опис - дупля не бью, вже тут все облазив, ніяк не можу знайти як це зробити.

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**